modules/introduction/partials/new-features-72.adoc

```diff
@@ -0,0 +1,8 @@
+* _Per Collection Change-History_.
+A _change history_ can be maintained for any collection except the `_default` collection.
+Every change made to a document within the collection is included in the change history.
+A maximum size for the change history can be specified; in bytes, and also in seconds.
```

Comment:
Possibly clarify that the size of history (bytes/seconds) is for the bucket and not something defined for each collection?

Reply:
Will do.